services/authorization/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v0.1.2 (2024-04-09)
+
+- Removed unused models.
```

Comment:
Is "model" a common word for a struct/type?

Reply:
added the word data type